ramalama/model.py

```diff
@@ -118,9 +118,9 @@ def _image(self, args):
 
         if args.runtime == "vllm":
             if gpu_type == "HIP_VISIBLE_DEVICES":
-                return "quay.io/modh/vllm:rhoai-2.17-rocm"
+                return "quay.io/modh/vllm:rhoai-2.18-rocm"
```

Comment:
Would it make sense to have such values configurable in a dedicated .conf file for ramalama? For example, having 
`VLLM_CONTAINER_CPU=quay.io/modh/vllm:rhoai-2.18-rocm`
`VLLM_CONTAINER_GPU=quay.io/modh/vllm:rhoai-2.18-cuda`

Reply:
We started something like that, if you grep around the codebase you'll see it:

ramalama.conf

to keep it simple we map things directly to the CLI args (but a CLI arg would take precedence over a conf setting). But I haven't used it in a long time, feel free to test, edit, enhance that config file.